src/services/sync/repo/user.repo.js

```diff
@@ -117,8 +117,8 @@ const findByLdapIdAndSchool = async (ldapId, schoolId) =>
 const findByLdapDnsAndSchool = async (ldapDns, schoolId) =>
 	userModel
 		.find({
-			ldapDn: { $in: ldapDns },
 			schoolId,
+			ldapDn: { $in: ldapDns },
```

Comment:
Does the order here make a difference?

Reply:
shoud be, i hope so thate the B-Tree Build Structur is a better.